cluster/test-deploy/openshift-ansible-aws-atomic/vars-origin.yaml

```diff
@@ -0,0 +1,119 @@
+ansible_become: true
```

Comment:
The name of this profile should be aws-atomic, not openshift-ansible-aws-atomic

Reply:
Renamed. The other profile has been renamed in https://github.com/openshift/release/pull/1582